tests/phpunit/CRM/Core/Payment/AuthorizeNetIPNTest.php

```diff
@@ -270,7 +270,8 @@ public function testIPNPaymentMembershipRecurSuccessNoLeakage() {
       'Supporter Profile',
       'First Name: Anthony',
       'Last Name: Anderson',
-      'Email Address: anthony_anderson@civicrm.org',
+      'Email Address:',
```

Comment:
This doesn't look good - if this change is affecting this test that is a red flag